R/deps.R

```diff
@@ -167,17 +167,7 @@ parse_one_remote <- function(x) {
       envir = asNamespace("devtools"), mode = "function", inherits = FALSE),
     error = function(e) stop("Unknown remote type: ", type, call. = FALSE))
 
-  simple_remote <- fun(repo)
```

Comment:
Yeah removing this looks right, I just ran into this as well when debugging Kirills' dtplyr issue.